### PR TITLE
People: Ensure sign in link shows on logged out flow

### LINKED ISSUE #363: People: Ensure sign in link shows on logged out flow
At `/start/account/user`, there is a link at the bottom of the form that says "Already have a WordPress.com account? Log in now."

We'll need something similar for the logged out invite accept flow. Clicking that link should probably take the user to WordPress.com to login. After the user logs in, we can redirect them back to the logged in flow.

### PR DESCRIPTION
**Pull request by lezama**
image

### How to test ###

* pull `add/invites-logged-out-log-in-footer`
* go to `http://calypso.dev:3000/accept-invite/<blog_id>/<invitation_key>`
* test the footer link :)

Closes #363

cc @ebinnion

### CODE REVIEW

File: client/accept-invite/logged-out-invite/signup-form.jsx [hunk 1]
```diff
@@ -11,11 +10,7 @@ import SignupForm from 'components/signup-form'
 import InviteFormHeader from '../invite-form-header'
 import { createAccount, acceptInvite } from '../actions'
 import WpcomLoginForm from 'signup/wpcom-login-form'
-
-/**
- * Module variables
- */
-const debug = debugModule( 'calypso:accept-invite:logged-out' );
```
Comment (ebinnion) [inline client/accept-invite/logged-out-invite/signup-form.jsx]: Instead of getting rid of `debug` here, you could actually use it :stuck_out_tongue:. The logic around accepting an invite would be ideal. Not a blocker though.

File: client/accept-invite/logged-out-invite/signup-form.jsx [hunk 2]
```diff
@@ -103,6 +98,15 @@ export default React.createClass( {
 		)
 	},
 
+	footerLink() {
+		let logInUrl = config( 'login_url' );
+		if ( config.isEnabled( 'login' ) ) {
+			logInUrl = '/log-in';
+		}
+		logInUrl += '?redirect_to=' + encodeURIComponent( window.location.href );
+		return <a href={ logInUrl } className="logged-out-form__link">{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }</a>;
```
Comment (ebinnion) [inline client/accept-invite/logged-out-invite/signup-form.jsx]: I don't believe that we're properly component-izing the CSS here. If I recall, this is how it's done in the signup framework as well.

We should fix how we handle this, although it could be in a different PR.

File: client/accept-invite/logged-out-invite/signup-form.jsx [hunk 3]
```diff
@@ -103,6 +98,15 @@ export default React.createClass( {
 		)
 	},
 
+	footerLink() {
+		let logInUrl = config( 'login_url' );
+		if ( config.isEnabled( 'login' ) ) {
+			logInUrl = '/log-in';
```
Comment (ebinnion) [inline client/accept-invite/logged-out-invite/signup-form.jsx]: This doesn't work for me on my a8c account with 2fa enabled. I think we should remove this conditional, including lines `103-105`.

File: client/accept-invite/logged-out-invite/signup-form.jsx (hunk 2 above)
Comment (ebinnion) [inline client/accept-invite/logged-out-invite/signup-form.jsx]: Can we reformat this a bit so it fits better? Perhaps:

```
return (
	<a href={ logInUrl } className="logged-out-form__link">
		{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }
	</a>
);
```

**Comment by ebinnion** [comment]
Changing label to awaiting fixes because the login URL doesn't work for me in development.

**Comment by lezama** [comment]
> Changing label to awaiting fixes because the login URL doesn't work for me in development.

it doesn't work on development for me neither, there's a js error when trying to finis the login flow: 
image

cc @scruffian

**Comment by lezama** [comment]
Removed the usage of `/log-in`, it does work on development now.

**Comment by ebinnion** [comment]
LGTM. No comments on code and functionally this works well. :shipit:
